log/src/main/java/io/airlift/log/Logger.java

```diff
@@ -74,6 +75,7 @@ public void debug(Throwable exception, String message)
      *
      * @param message a literal message to log
      */
+    @FormatMethod
```

Comment:
does `log.debug("%%")` log single percent or two?

Reply:
is the caller allows to call `log.debug("%...a malformed format string")`?

Reply:
Ah, I haven't thought of that :/ No, the `%`s won't be interpreted by these methods, but Error Prone will treat them as format strings regardless. So, back to the drawing board I guess.

Reply:
I pushed an update, with the following caveat:

```
(...) from now on the message on these methods will be actually treated as format string (...) This is a breaking change, but it makes treatment of the messages consistent across all variants of these log methods.
```

Are we willing to accept that?